packages/apps/transfer/src/pages/transfer/module-explorer/networks/[network]/chains/[chain]/modules/[module]/styles.ts

```diff
@@ -0,0 +1,39 @@
+import { styled } from '@kadena/react-components';
```

Comment:
`styled` doesn't sound very vanilla extract-y? Isn't this the deprecated component lib? @eileenmguo @sstraatemans

Reply:
Correct, legacy, is alreqady being fixed in another task